### DISCUSSION TITLE
Error in loading model

### DISCUSSION THREAD

**Discussion by f85r** [Trusted-AI/adversarial-robustness-toolbox#1661] [Q&A] [answered]
ModuleNotFoundError: No module named 'art.preprocessing.standardisation_mean_std.standardisation_mean_std'，could not find module

**Comment by f85r** [comment]
![image](https://user-images.githubusercontent.com/104835216/166482969-8e082e59-9519-4894-ae40-4ac92289d7c1.png)
like this

**Reply by beat-buesser** [reply]
Hi @f85r Can you share a code snippet that produces this error?

**Reply by f85r** [reply]
![image](https://user-images.githubusercontent.com/104835216/166652013-8acbcbd7-9529-48d9-a5d6-376dd549ce8e.png)
E:/bspic/classifier_advanced_14 - 副本.txt is  model built by tensorflow and  KerasClassifier

**Reply by f85r** [reply]
from keras.utils.generic_utils import CustomObjectScope
        from art.preprocessing.standardisation_mean_std import utils
        with CustomObjectScope({'MyLayer': MyLayer}):
            classifier2 = pickle.load(open('E:/bspic/classifier_advanced_14 - 副本.txt', 'rb'))

**Reply by beat-buesser** [reply]
Thank you. How did you create the file `classifier_advanced_14 - 副本.txt`?

**Reply by f85r** [reply]
classifier_advanced_14 - 副本.txt is built by others, he gave me

**Comment by f85r** [comment]
[classifier_advanced_14.txt](https://github.com/Trusted-AI/adversarial-robustness-toolbox/files/8628448/classifier_advanced_14.txt)

**Reply by f85r** [reply]
this is the txt

**Reply by beat-buesser** [reply]
Can you share how you have created it?

**Comment by f85r** [comment, marked as answer]
yes


2022-05-05 21:24:54"Beat Buesser" ***@***.***>写道：

Can you share how you have created it?

**Reply by f85r** [reply]
I have already solve this question ,because of the Version mismatch. Thank you!
